app/assets/javascripts/discourse/app/lib/user-search.js

```diff
@@ -182,12 +181,9 @@ export default function userSearch(options) {
 
     cacheKey = newCacheKey;
 
-    const clearPromise = later(
-      () => {
-        resolve(CANCELLED_STATUS);
-      },
-      isTesting() ? 250 : 5000
-    );
+    const clearPromise = later(() => {
```

Comment:
This `clearPromise` seems like a custom way to cancel the search if it takes too long. I think it might be better to not execute this code at all in test mode rather than give it 5 seconds:

```js
let clearPromise;                                             
if (!isTesting()) {                                           
  clearPromise = later(() => resolve(CANCELLED_STATUS), 5000);
}                                                             
```

thoughts?

Reply:
Yes probably better indeed, chances a search timeouts in test seem low and probably something we could fix